translations/Readme.md

```diff
@@ -0,0 +1,138 @@
+# About Translation
+
+## Files list
+
+- [Windows](#windows)
+
+    ````text
+    lupdate.cmd
+    lrelease.cmd
+    ````
+
+- [macOS / Linux](#macos-linux)
+
+    ```` text
+    translation_utility.sh
+    ````
+
+## How to use the translation utilities
+
+### Requirement
+
+- These utilities needs [QT](https://qt.io) library.
+
+### On Windows {#windows}
+
+#### 1. Adding translation `(.ts)` files
+
+1. Open `lupdate.cmd` in editor. (e.g, VSCode or Notepad, etc.)
+2. Append your new file name to the end of the first line, in the format **locale_**_language-code_**.ts** (e.g, locale_de, local_zh-cn, etc.).
+3. Double click on `lupdate.cmd` or run it in `cmd.exe`.
+4. You will find the new file in the same directory as the `lupdate.cmd`.
+
+#### 2. Updating `.qm` files
+
+1. Just double click on `lrelease.cmd` or run it in `cmd.exe`.
+2. You will find the new file in the same directory as the `lrelease.cmd`.
+
+#### 3 Start Translating
+
+1. Right click on a `.ts` fils to open with `Qt Linguist.exe`
+2. Start translating phrases.
+
+#### 4. Preview Your Translation
+
+Copy and move the whole `batch_export` folder to  MuseScore `plugin` (You may check the path in MuseScore.exe, from the `main menu`: `File` &rarr; `Preferences...` &gt; `General` tab &gt; `Folders` section) to test the translation. 
+You may need to adjust some phrases during this process. Then do step 2 and step 3 again.
+
+### On Linux / macOS {#macos-linux}
+
+Use `translation_utility.sh`.
+
+This is a 2-in-1 utility, which enables you to process `.ts` and `.qm` files in an efficient way.
+
+#### 1. Preparation
+
+Before running the script, you have to assign privilege for `translation_utility.sh` using
```

Comment:
Not sure this is needed, as far as I can tell it has the eXecute but set on GitHub already? Surely won't harm so set it though

Reply:
This is for some mac volunteers, such as me. ￼

Reply:
As said, it surely won't harm to set it